src/System.Private.Xml/src/System/Xml/XPath/Internal/CacheChildrenQuery.cs

```diff
@@ -92,10 +92,12 @@ public override XPathNavigator Advance()
                         continue;
                     }
                 }
-				// TODO: this code was previously removed
+                // TODO: this code was previously removed
 #if DEBUG
```

Comment:
>if DEBUG [](start = 1, length = 8)

I would delete this #if DEBUG. It looks like temporary code for some VS bug investgiation. And the set of _lastNode only in debug is worrying.